forge-gui/res/cardsfolder/upcoming/jolene_the_plunder_queen.txt

```diff
@@ -0,0 +1,16 @@
+Name:Jolene, the Plunder Queen
+ManaCost:2 R G
+Types:Legendary Creature Human Warrior
+PT:2/2
+
+T:Mode$ AttackersDeclared | AttackedTarget$ Opponent | Execute$ TrigTreasure | TriggerZones$ Battlefield | TriggerDescription$ Whenever a player attacks one or more of your opponents, that attacking player creates a Treasure token.
+
+SVar:TrigTreasure:DB$ Token | Defined$ AttackingPlayer | TokenScript$ c_a_treasure_sac | TokenOwner$ TriggeredAttackingPlayer
+
+R:Event$ CreateToken | ActiveZones$ Battlefield | ValidPlayer$ You | ValidToken$ Treasure | ReplaceWith$ DBReplace | Description$ If you would create one or more Treasure tokens, instead create those tokens plus an additional Treasure token.
+SVar:DBReplace:DB$ ReplaceToken | Type$ AddToken | Amount$ 1 | ValidCard$ Treasure | TokenScript$ c_a_treasure_sac
+DeckNeeds:Type$Token

```

Comment:
Not needed since this card generates tokens by itself.

Reply:
Just checking, it's the "DeckNeeds:Type$Token" that's not necessary here?

Reply:
Yep. DeckNeeds specifies a certain condition that's absolutely necessary for this card to function in the deck. This card is able to generate Treasure tokens by itself, so it doesn't absolutely need another token generator.

Reply:
Ah, I see. Thanks!